src/paging/index.ts

```diff
@@ -23,10 +23,12 @@ class Paging {
   };
 
   init() {
+    this.updatePage();
```

Comment:
Instead of always calling `this.select` in updatePage, leave that function as is and call `this.updatePage(this.select(Selectors.pageObject)));` here

Reply:
agree